.github/workflows/maven.yml

```diff
@@ -18,7 +18,7 @@ jobs:
       max-parallel: 5
       matrix:
         os: [ubuntu-latest, windows-latest]
-        java: [8, 11, 17]
+        java: [11, 17, 21]
```

Comment:
Looks like you will need to update the Maven version to at least 3.8.6. We are using the following for LMP:

```
- name: Set up Maven
      uses: stCarolas/setup-maven@v4.5
      with:
        maven-version: 3.9.2
```